src/components/routes/Trips/Trips.tsx

```diff
@@ -59,77 +60,111 @@ class Trips extends React.Component<Props, State> {
+              const isUpcomingEmpty = !(upcoming || []).length;
               return (
                 <>
-                  {isSubmitting && <CryptoPortal />}
-                  {!!upcoming.length && (
-                    <section>
-                      <AppConsumer>
-                        {({ screenType }: AppConsumerProps) => {
-                          const mobile = screenType < ScreenType.TABLET;
-                          return (
+                  <TabNavBar config={[
+                    {
+                      title: 'Current',
+                      to: '/trips/current',
+                    },
+                    {
+                      title: 'Upcoming',
+                      to: '/trips/upcoming',
+                    },
+                    {
+                      title: 'Past',
+                      to: '/trips/past',
+                    },
+                    {
+                      title: 'Cancelled /\nRejected',
+                      to: '/trips/cancelled',
+                    }
+                  ]} />
+                  <div className="trip-cards-container">
+                    <Switch>
+                      <Route exact path="/trips/current" component={() =>
+                        <>
+                          {!!started.length &&
                             <>
-                              {!mobile && <h3>Upcoming Trips</h3>}
-                              <div className="active-cards-container">
+                              <div className="started-trip-container">
+                                <h3>Finish this booking:</h3>
+                                <StartedTripCard
+                                  trip={started[0]} />
+                              </div>
+                              <Divider size="tall" />
+                              <h2>Current bookings:</h2>
+                            </>
+                          }
+                          {!!current.length &&
+                            <section className="active-cards-container">
+                              {current.map((trip: Booking) =>
+                                <ActiveTripCard 
+                                  onCancelClick={this.handleCancelBooking.bind(this, trip)}
+                                  key={trip.id}
+                                  trip={trip} />
+                              )}
+                            </section>
+                          }
+                        </>
+                      } />
+                      <Route exact path="/trips/upcoming" component={() =>
+                        <>
+                          {isUpcomingEmpty
+                            ?
+                              <div className="trips-book-now">
+                                <div className="trips-book-now--text">
+                                  <h2>You haven't booked any trips yet.</h2>
+                                  <BeeLink to="/">Let's change that!</BeeLink>
```

Comment:
i dont think we need two links, maybe get rid of the Let's change that link and keep the button?

Reply:
sg. removed `Let's change that!`